src/libs/antares/object/intrusive-reference.h

```diff
@@ -28,7 +28,7 @@
 #define __ANTARES_LIB_ASSEMBLY_INTRUSIVEREFERENCE_H__
 
 #include "../antares.h"
-#include <yuni/core/smartptr.h>
+#include <memory>
```

Comment:
no need to include **memory** here, it complies just fine without it

Reply:
done